### ISSUE TITLE
bug in (Varying modal content)

### ISSUE THREAD

**Issue by kkpatel911** [twbs/bootstrap#27959]
[https://getbootstrap.com/docs/4.0/components/modal/](url)
![bug](https://user-images.githubusercontent.com/25020357/50577125-c3ced380-0dee-11e9-8a1d-c2b2a59c373a.JPG)

- Operating system and version (Windows 10)

**Comment by Johann-S** [comment]
I don't see any issues @kkpatel911 can you add some details please

**Comment by bardiharborow** [comment]
@kkpatel911 is your concern that our button components do not have top and bottom padding on them?

**Comment by kkpatel911** [comment]
> @kkpatel911 is your concern that our button components do not have top and bottom padding on them?

Yes, i didn't know how to search for that file  since i am new. but if u add padding or margin to that so the button would look nice in smaller screen.

**Comment by florianlacreuse** [comment]
@kkpatel911 See https://github.com/twbs/bootstrap/blob/v4-dev/site/docs/4.2/components/modal.md#varying-modal-content
